packages/subscriptions/lib/Subscriptions.js

```diff
@@ -5,6 +5,19 @@ const {
   isUserUnrestricted,
   includesUnrestrictedChildRepoId
 } = require('@orbiting/backend-modules-documents/lib/restrictions')
+const uniq = require('lodash/uniq')
+const { ascending } = require('d3-array')
+const { parse, Source } = require('graphql')
+const schemaTypes = require('../graphql/schema-types')
+
+const EventObjectTypes = parse(new Source(schemaTypes))
```

Comment:
Here is a crazy idea, something we haven't done before but was thinking about for quite a while now:

Define constants (in this case types) in lib file and then export them into `lib/schema-types.js`? Later is very much (and maybe purposefully) a JavaScript file and feels straighter to me.

`schema-types.js` might look like this:


```js
const EVENT_OBJECT_TYPES = require('/lib/Something.js')
const SUBSCRIPTION_OBJECT_TYPES = require('/lib/Buzzfizz.js')

module.exports = `

enum EventObjectType {
${EVENT_OBJECT_TYPES.join('\n')}
}
union EventObject = Comment | Document

enum SubscriptionObjectType {
${SUBSCRIPTION_OBJECT_TYPES.join('\n')}
}
union SubscriptionObject = ${SUBSCRIPTION_OBJECT_TYPES.join(' | ')} | Discussion

...

`

```

Reply:
As it's done like this in other places, I'd rather not go into this now.